service/storage_proxy.cc

```diff
@@ -2446,7 +2446,11 @@ void storage_proxy::got_failure_response(storage_proxy::response_id_type id, gms
 void storage_proxy::maybe_update_view_backlog_of(gms::inet_address replica, std::optional<db::view::update_backlog> backlog) {
     if (backlog) {
         auto now = std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count();
-        _view_update_backlogs[replica] = {std::move(*backlog), now};
+        auto timestamped_backlog = view_update_backlog_timestamped{*backlog, now};
+        auto[it, inserted] = _view_update_backlogs.try_emplace(replica, timestamped_backlog);
+        if (!inserted) {
+            it->second = timestamped_backlog;
```

Comment:
insert_or_assign()

Reply:
Good to know such a method exists, I used it in the rebase